src/stackmap_checker/stmap.c

```diff
@@ -72,14 +73,14 @@ uint64_t stmap_get_location_value(stack_map_t *sm, location_t loc,
         case REGISTER:
             return regs[loc.dwarf_reg_num];
         case DIRECT:
-            dest_addr = (uint64_t *)malloc(sizeof(uint64_t));
+            dest_addr = (uint64_t *)calloc(1, sizeof(uint64_t));
```

Comment:
Ah, sorry, I should have said. There's no need to use `calloc` if you are allocating 1 of something. Only if you would be forced to otherwise use multiplication.